src/sorted_run_iterator.rs

```diff
@@ -179,7 +186,14 @@ mod tests {
         let root_path = Path::from("");
         let object_store: Arc<dyn ObjectStore> = Arc::new(InMemory::new());
         let format = SsTableFormat::new(4096, 3, None);
-        let table_store = Arc::new(TableStore::new(object_store, format, root_path.clone()));
+        let block_cache = Some(Arc::new(BlockCache::new((1 << 30) / 4096))); // 1GB block cache,
```

Comment:
nit: 

there's been so many occurrences of this computation `(1 << 30) / 4096` which yields a constant value... may  be there should be a declared constant which is then reused. easy to change or make it an optional flag/param (w/ default fallback being the constant) later on.